soroban-sdk/src/env.rs

```diff
@@ -469,6 +469,79 @@ use std::{path::Path, rc::Rc};
 #[cfg(any(test, feature = "testutils"))]
 use xdr::{LedgerEntry, LedgerKey, LedgerKeyContractData, SorobanAuthorizationEntry};
 
+#[cfg(any(test, feature = "testutils"))]
+pub struct TestStellarAssetContractUtil {
+    pub address: Address,
+    env: Env,
+    issuer_id: xdr::AccountId,
+}
+
+#[cfg(any(test, feature = "testutils"))]
+impl TestStellarAssetContractUtil {
+    pub fn get_issuer_flags(&self) -> u32 {
```

Comment:
Can consts be added in a mod for the values of the flag?

Reply:
I exposed `xdr::AccountFlags` as  `IssuerAccountFlags`.

Reply:
The `AccountFlags` type is `repr(i32)` (enums are signed integers in XDR [ref](https://datatracker.ietf.org/doc/html/rfc4506#section-4.3)) not `u32`. We should update the type to match.

Reply:
I updated the PR, but I just realized that the issuer flag field in xdr is a u32, so my update is actually inconsistent now. There's going to be some inconsistency regardless of what we do here. I think `flags()` should return u32, and `overwrite_issuer_flags()` should take a u32. What do you think?

Reply:
Ah yeah. 👍🏻